### PR TITLE
Add sunshine team final plog post

### PR DESCRIPTION
**Pull request by ianchobuilds**
Co-Authored-By: Sujin Lee sujinlee.me@gmail.com

### CODE REVIEW

File: blog/_posts/2018-09-27-sunshine-team-goodbye.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+created_at: Thurs September 27 2018
+permalink: blog/2018-09-27-sunshine-team-goodbye
+current: blog
+author: Gyujin and Sujin
+categories:
+- student-posts
+---
+
+__Time flies ✈️__
+We could not be happier with the decision we made to apply for RGSoC and cannot believe it is time to say goodby already. This summer has been literally amazing.
```
Comment (anasofiapinho) [inline blog/_posts/2018-09-27-sunshine-team-goodbye.md]: goodby -> goodbye

File: blog/_posts/2018-09-27-sunshine-team-goodbye.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Reading tons of open source codes also helped us a lot. This part actually took the most of our time and was the hardest part. Add to the  Babel document site with [Repl](https://babeljs.io/repl) which we worked on and Babel Core, we had to go through [Docusaurus](https://docusaurus.io/)  since Babel document site is built on it, [ASTExplorer](https://astexplorer.net/), [Redux Devtools](https://github.com/zalmoxisus/redux-devtools-extension), [CodeSandbox](https://codesandbox.io/) and other several JSON tree view libraries. Taking time to understand various projects helped us a lot with JS project structures, how these wonderful open source products are implemented and learned how others approach problems. We got able to grasp others’ code quicker, speed up our code reading skills. :) 
+
+Of course, we got to know better how Babel works inside. We studied about AST(Abstract Syntax Tree) and Babel core. It was also our first time to use CSS-in-JS  and flow, static type checking system for JavaScript.
+
+__But the most important thing we learned is OPEN SOURCE SPIRIT!__
```
Comment (anasofiapinho) [inline blog/_posts/2018-09-27-sunshine-team-goodbye.md]: is -> was the OPEN SOURCE SPIRIT

File: blog/_posts/2018-09-27-sunshine-team-goodbye.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+This summer was a great opportunity for Sunshine team to learn about not only actual technical skills, but also *open source spirit — improving by cooperation, not competition.* Friends in Babel community and RGSoC were so nice and generous, they helped us a lot with the code and also constantly found good aspects in our works. We learned how to stay positive and thoroughly evaluate others’ work with warmth — the most valuable asset to keep working as a developer. It really helped us to be confident. Now we have no fear to dive into the open source world!
```
Comment (anasofiapinho) [inline blog/_posts/2018-09-27-sunshine-team-goodbye.md]: but also *the* open source spirit

File: blog/_posts/2018-09-27-sunshine-team-goodbye.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+__After RGSoC__
+* Keep workin’ on 
+We know that it would not be easy but we still hope we can keep contributing to Babel Project. Of course, we want to improve [our first merged PR, Time Travel UI]() and also AST output panel we worked lately.  Other than that, someday we want to implement Redux or other state management system! :)
```
Comment (anasofiapinho) [inline blog/_posts/2018-09-27-sunshine-team-goodbye.md]: to *the* Babel Project

File: blog/_posts/2018-09-27-sunshine-team-goodbye.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+* [Sujin](https://twitter.com/sujinleeme) will continue to pursue her Master’s Degree study in Music and Engineering. 
+* [Gyujin](https://twitter.com/MarinaGJCho) is going to keep working as a frontend engineer. 
+
+__Thanks to__
+
+__Dear RGSoC team!__
+We cannot say thank you enough to our super kind and lovely supervisor, [Ana Sofia Phino](https://twitter.com/anasofiapinho)! She was always kind and helping, our team relied on her so much. We cannot imagine our RGSoC without her and so sad to say goodbye. 😢 We’d also like to thank [Saptak](https://twitter.com/Saptak013) who helped us to keep on track when Ana was on vacation, [Vaishali](https://twitter.com/kernel_girl) and [Ramon](https://twitter.com/senorhuidobro) who interviewed us and gave us this great chance. Our team also had a fantastic opportunity to meet Ramon in person when he visited Seoul for holidays! We hope we could meet other RGSoC members too!
```
Comment (anasofiapinho) [inline blog/_posts/2018-09-27-sunshine-team-goodbye.md]: Pinho ;)

**Comment by anasofiapinho** [comment]
Hey @gyujincho, I reviewed the post and it probably didn't even need my review! I just would made some minor suggestions, other than that I think it's good to go. Congrats and thank you! I really loved being part of Sunshine and I hope to see both of you soon online or offline!

**Comment by v-thakkar** [comment]
Hi @gyujincho ! Do you plan to submit changes Ana has suggested?

**Pull request merged**